src/Command/AbstractCommand.php

```diff
@@ -23,7 +23,7 @@
 use Symfony\Component\Console\Style\SymfonyStyle;
 use function Symfony\Component\String\u;
 use Symfony\Component\Yaml\Yaml;
-
+use Symfony\Component\String\UnicodeString;
```

Comment:
Is this needed?

Reply:
It's not needed. I run php-cs-fixer but some error from other files ddint allow fix it.

Reply:
You add properties type which is allowed from PHP 7.4, I try use 7.2 :smile: